pyproject.toml

```diff
@@ -21,12 +21,17 @@ fastapi = "0.97.0"
 uvicorn = "0.22.0"
 influxdb-client = "1.37.0"
 langid = "1.1.6"
-torch = {version = "^2.0.1+cpu", source = "pytorch"}
+torch = [
```

Comment:
It would be nice to make the change for mac support in separate PR.

Reply:
Let's keep the change because it's really small.